### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by MDVAurelien**
Add some translations

### CODE REVIEW

File: src/assets/i18n/fr.po [hunk 1]
```diff
@@ -419,7 +419,7 @@ msgid ""
 msgstr ""
 "Type de<a "
 "href=\"https://www.fastmail.com/help/technical/ssltlsstarttls.html\" "
-"target=\"_blank\"> chiffrement des courriels</a>. Les choix sont <i>Plain "
+"target=\"_blank\"> chiffrement des emails</a>. Les choix sont <i>Plain "
```
Comment (denysbutenko) [inline src/assets/i18n/fr.po:422]: Below all words have a dash `e-mails`, is it a typo?

Comment (MDVAurelien) [reply src/assets/i18n/fr.po:422]: No in France it is more conventional to read 'E-mail' but both are valid. It is also common to find the term 'courriel' in the Administration. Thanks

**Comment by kmoore134** [comment]
ok to test

**Review by denysbutenko — APPROVED** [review]
LGTM

**Pull request merged**